cem/support-typedef-jsdoc-utils.js

```diff
@@ -1,70 +1,143 @@
 import { readFileSync } from 'fs';
 import path from 'path';
 
+export function getTypesFromClass (classNode, ts, retrievePrivate = false) {
+  const result = [];
```

Comment:
nitpick (non-blocking): we return an array of types, maybe we could rename this var as `types` to improve consistency. It looks like we use `types` or `xTypes` elsewhere in the code when we return some types.